build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 
 allprojects {
     group = "com.github.streem.pbandk"
-    version = "0.8.2-SNAPSHOT"
+    version = "0.8.21-SNAPSHOT"
```

Comment:
0.8.2 is already the version of the upcoming release. You don't need to bump the version number unless your PR is introducing breaking changes that require us to bump it to 0.9.0. Otherwise, once we're ready to do a release we'll just bump the version from `0.8.2-SNAPSHOT` to `0.8.2`, do the release, and then bump the version to `0.8.3-SNAPSHOT`.